src/oas.md

```diff
@@ -1869,6 +1878,30 @@ requestBody:
 
 As seen in the [Encoding Object's `contentType` field documentation](#encoding-content-type), the empty schema for `items` indicates a media type of `application/octet-stream`.
 
+###### Example: Nested `multipart/mixed`
+
+This defines a two-part `multipart/mixed` where the first part is JSON and the second part is a nested `multipart/mixed` document.
+The nested parts are JSON, plain text, and a PNG image.
+
+```yaml
+multipart/mixed:
+  schema:
+    type: array
+    prefixItems:
+    - type: array
```

Comment:
Shouldn't the first prefix item be`type: object` and the second have `type: array` and `prefixItems`?

If not, comments in or more prose around the example would be helpful.

Reply:
The first one is correct, as I wanted to highlight that in ordered multipart, `type: array` defaults to `application/json` as opposed to the convoluted (but RFC-based) logic for arrays for `form-data`).  I have changed "is JSON" to "is a JSON array" in the text to clarify.

The second was missing `type`, and also the first nested entry is XML and not JSON, which I have corrected in the text.